crates/host/src/wasmbus/mod.rs

```diff
@@ -1604,15 +1649,16 @@ fn linkdef_hash(
     hex::encode_upper(hash.finalize())
 }
 
-#[instrument(level = "debug", skip_all)]
-async fn create_lattice_metadata_bucket(
+#[instrument(skip(jetstream, bucket_message_name))]
```

Comment:
Does this need to trace at the info level or can it stay debug?

Reply:
The rebase with your changes was a bit difficult so this got lost in the shuffle. Will change